### PR TITLE
Second attempt at dev-challenge-01

### PR DESCRIPTION
**Pull request by josephalsup**

### CODE REVIEW

File: docs/dev-profile/alsup-dev-profile.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title> Joseph's Dev Profile </title>
```
Comment (barrycumbie) [inline docs/dev-profile/alsup-dev-profile.html:4]: what's with these leading and trailing spaces? I'm tasking you to look at the Google Style guide and see what it says, then to reply and enact what it says throughout this page. 

```diff
- <title> Joseph's Dev Profile </title>
+ <title>Joseph's Dev Profile</title>
```

File: docs/dev-profile/alsup-dev-profile.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
...tDn9MHip8iIhRQZDQlNKDZg2LTf+flXzkZKZTjKSv5GbejlPlUDDExD8p9jkbV3KkBAG44R3a1o5RXEnrbc1RH218HeJbo9bmUqiLlDcVA6/kb47UTqndVuLb+t9ZUxScSwEnwJzYaaPUcW2KQqVr8MWl9V9K2a4/KmpoQc/tbcb6L7+yixj4/FTbjgaoK9X0b7ykgZE476tLzgfJcf/gIIZbjuqq0LLdHSW/3qmgQYnF0kTbmSsLHTp4IbyUsLDTsD5qdIg3+tQDDfFpur9Sg0K2TwJOEoKqeltFejmwIuuDr/skYrp0s2xJZ3wgixFBDEyoS6qmK34O1ZNez8Nzg+og8brptCM0DDbRMGaBlaIBjbRaYbaAM0ENtFyE0CBtEj7aCZlhER7IIPiM9GmatcfLbJTiY1VK+WK6Z4wqWqWW6GSmBVj6ssQzkWA/4N8RxGdT4bk9psnHenfz8qJqd7Wufz0baztAacMZTROVPDS2lssqi0yUqcqzKCzpZPMZdpZ1/St5jTVsezRR0hOVd1FfSYJOoIdDwkez20OlxDQKnUrIpcYGk1x/6Cy06Eu3ssUql0DwBZ2grCtRGGGb6DbYfUrOtEgqmhdyoETaGAbWdIey2iaYFBAgQIFFoI20XED6BXFII21PpIMbCs6AZtq9UDMFaBjBWgOJhEFOfWU0zAWcTAaqDwfQWUM2oCyhgVQMBtUhW1nD2QUUlkDaCa2gmBFwkEDwbQVWP5q5CNA60Ny/vOGrsTk7VuOYkn5E2ZOnw11Jl5vLnZKZZSIevsK/ll5VsTHUCLf61YhTPRG0oK1xIQEiuU/QQU+KsVVOENS08oMZMp5/2V3IrYZhxzw1o5ZVlLZVSQ2x+mruUGnhsIuOEK+oOkSfCItZqUF9hjEYsO1U5H7jDjfhOM2lnywummenE4iYYibiWwcDWe825wkxQoumYmT7Duap5jw/5161K2e5rT0TsN7T99aRaqplKs1Wp9uzaU02GnEc00NLrgeW3+9QVGKZMQDDvtVuS7bct/bPeMoJtThvA5i/DvtOX4H5HPuUTl3iQsLnU7eKG5luUTJZqag1/acdVIoTfJ1z/AINqpoeqNF2gi1BjHOEGfo2/C4ytcnytnDP95DlMzMaUTHsnVmQPZfpqVQyWKsY5rUMNx/ht7rxnkH//2Q==" alt="Picture of me">
```
Comment (barrycumbie) [inline docs/dev-profile/alsup-dev-profile.html:12]: need to fix this...base64 encoding of an image.

File: docs/dev-profile/alsup-dev-profile.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+		<h4> <a target="_blank" href="https://github.com/josephalsup"> GitHub Profile </a> </h4>
+		<h4> <a target="_blank" href="https://validator.w3.org/nu/?doc=https%3A%2F%2Fjosephalsup.github.io%2Fdev-challenge-01%2Fdev-profile%2Falsup-dev-profile.html"> Nu Checker </a> </h4>
```
Comment (barrycumbie) [inline docs/dev-profile/alsup-dev-profile.html:15]: this link will actually need to point to this pages address ONCE IT GETS merged...so to barrycumbie.github.io

kindly give me this Nu link to your active test page in the PR comments !

File: docs/dev-profile/alsup-dev-profile.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+		<h4> <a target="_blank" href="https://github.com/josephalsup"> GitHub Profile </a> </h4>
```
Comment (barrycumbie) [inline docs/dev-profile/alsup-dev-profile.html:14]: put your github username in the text of the link

**Review by barrycumbie — COMMENTED** [review]
You need to delete the index.html from this PR...should only be one file

File: docs/dev-profile/alsup-dev-profile.html (hunk 1 above)
Comment (josephalsup) [reply docs/dev-profile/alsup-dev-profile.html:4]: I see it now, the style guide says to remove trailing white spaces

**Pull request closed without merging**